molo/profiles/tests/test_views.py

```diff
@@ -90,6 +90,10 @@ def test_login(self):
         response = self.client.get(reverse('molo.profiles:auth_login'))
         self.assertContains(response, 'Forgotten your password')
 
+    def test_login_success(self):
```

Comment:
@Ntu2koTrevor you need to test here if a user logs in it will redirect them to the success page